checkov/kubernetes/checks/resource/k8s/KubeletClientCa.py

```diff
@@ -7,7 +7,7 @@
 class KubeletClientCa(BaseK8sContainerCheck):
     def __init__(self) -> None:
         # CIS-1.6 4.2.3
-        id = "CKV_K8S_140"
+        id = "CKV_K8S_C140"
```

Comment:
```suggestion
        id = "CKV_K8S_140"
```
I think this was not intended 🙂